bundles/org.openhab.core.ui/src/main/java/org/openhab/core/ui/internal/components/UIComponentSitemapProvider.java

```diff
@@ -220,6 +220,10 @@ protected Sitemap buildSitemap(RootUIComponent rootComponent) {
                 setWidgetPropertyFromComponentConfig(widget, component, "legend", SitemapPackage.CHART__LEGEND);
                 setWidgetPropertyFromComponentConfig(widget, component, "forceAsItem",
                         SitemapPackage.CHART__FORCE_AS_ITEM);
+                setWidgetPropertyFromComponentConfig(widget, component, "yAxisDecimalPattern",
```

Comment:
Can you comment on this? It seems missing before, why do we need it now?
(I am not familiar with the xchart properties)

Reply:
This solves a bug I discovered when working on this, see my PR description:

> Also fixes an issue where the `yAxisDecimalPattern` was not considered for managed sitemaps.

Apparently, this parameter was missed when creating the managed sitemap provider. The UI has support for it, but it never worked. I could remove this and put it in a separate PR, but it is only a one line fix.

Reply:
fine for me, thanks for the clarification